shaka/src/js/mse/video_element.cc

```diff
@@ -245,6 +247,8 @@ ExceptionOr<void> HTMLVideoElement::SetSource(const std::string& src) {
   if (src.empty())
     return {};
 
+  SetPlaybackRate(default_playback_rate_);
```

Comment:
I think technically this should go into the `Load` method.

Reply:
I feel like I had a reason for putting it here, but I can't seem to recall. In any case, I'll adjust it. :+1: